src/main/java/com/boot/user/model/CustomerMessage.java

```diff
@@ -25,19 +27,29 @@ public class CustomerMessage {
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
+    @Size(min = 3, message = "Min Last name size is 3 characters!")
+    @Size(max = 30, message = "Max Last name size is 30 characters!")
     @Column
     private String name;
 
-    @Size(min = 3, max = 50)
+    @Email(message = "Invalid Email!")
```

Comment:
You should keep also the size validation in order not to fail persist when using longer email